### PR TITLE
Make MVP bullet list; Add Motivation section

### PR DESCRIPTION
**Pull request by ystamaritq**
- Made MVP section a bullet list
- Added Motivation section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,20 +55,77 @@ Phase one design features basic functionality and style for multiple features.
...
+
+   For a reasonable fee Taylor was able to advertise to anyone in the area the exciting opportunity to come to his fair. Despite not having everything he wanted, the advertisement was very profitable and brought in many people who were unaware of the faire. With the extra money he made on his advertisement he was able to open again next year and move into a better house with his husband and their new child.
+
+1. Margret was incredibly bored. She had spent her whole highschool life living on cyclical ups and downs with nothing to do on the weekend. In desperation she found a Travel Board. Within under a minute she found a new opportunity to use her brand new driver's license to get out to an unknown music festival.
+
+   With such an exciting prospect she was able to coax her friends out of the house for a fun night. Despite staying up way too late, this experience was meaningful to her. For once she was able to express her own will, and desire to have fun on her own terms. Now Margret is able to express herself socially, and knows many of the exciting upcoming events in her town.
```
Comment (JDMartinez1531) [inline README.md:115]: Kind of a lot of these stories, but more content is better at this stage, right? Maybe we can put each of them in their own spoiler?

File: README.md [hunk 2]
```diff
@@ -76,6 +136,7 @@ Allow user to browse and save favorite destinations and more while providing wea
 ---
```
Comment (JDMartinez1531) [inline README.md:137]: Maybe put my name back on it?

Comment (ystamaritq) [reply README.md:112]: your name is there.

File: README.md (hunk 1 above)
Comment (ystamaritq) [reply README.md]: I removed the motivation part right. I think like Joshua. Willian can explain later his idea in a meeting and a summary can be added.

File: README.md (hunk 2 above)
Comment (ystamaritq) [reply README.md:112]: Make sure you expand the file. When there are no changes, github collapses those lines.

Comment (JDMartinez1531) [reply README.md:112]: Ok, sorry. I'm looking at everything on mobile so it's a little different.

**Pull request merged**